test-case/multiple-pause-resume.sh

```diff
@@ -36,10 +36,10 @@ OPT_NAME['r']='loop'     OPT_DESC['r']='pause resume repeat count'
 OPT_HAS_ARG['r']=1         OPT_VAL['r']=3
 
 OPT_NAME['i']='min'      OPT_DESC['i']='random range min value, unit is ms'
-OPT_HAS_ARG['i']=1         OPT_VAL['i']='100'
+OPT_HAS_ARG['i']=1         OPT_VAL['i']='70'
 
 OPT_NAME['a']='max'      OPT_DESC['a']='random range max value, unit is ms'
-OPT_HAS_ARG['a']=1         OPT_VAL['a']='200'
+OPT_HAS_ARG['a']=1         OPT_VAL['a']='120'
```

Comment:
Could  you please fix the header above or the OPT_DESC or both to explain what this mysterious "range" is?

Reply:
I updated the description also

Reply:
@fredoh9 usually I use "-i 20 -a 50 -l 100" locally to help to capture more issues within 2 hours, for daily, maybe "-i 20 -a 50 -l 20" could be good.

Reply:
@keyonjie do you recommend -i 20 -a 50 as default values?

Reply:
> @keyonjie do you recommend -i 20 -a 50 as default values?

yes, and run more loop (default 3 -> 20?) to capture issues more stably.

Reply:
Thank you for your suggestion, I think good point. But default loop/repeat value should be carefully decided. Daily test use "/multiple-pause-resume.sh -r 25" if default loop is 20, this will take too much. If any one want to run more thorough test, they can specify that count with -l/-r options.
I slightly increased loop/round count.

Reply:
sounds good.